### ISSUE TITLE
TimeSeriesScalerMinMax gives NaN when all values are the same

### ISSUE THREAD

**Issue by GillesVandewiele** [tslearn-team/tslearn#314]
**Describe the bug**
Scaling time series with `TimeSeriesScalerMinMax` results in NaNs, while this is not the case with `TimeSeriesScalerMeanVariance`.

**To Reproduce**
```python
>>> from tslearn.preprocessing import TimeSeriesScalerMeanVariance, TimeSeriesScalerMinMax
>>> scaler = TimeSeriesScalerMinMax()
>>> scaler.fit_transform([[0, 0, 0, 0]])
array([[[nan],
        [nan],
        [nan],
        [nan]]])
>>> scaler = TimeSeriesScalerMeanVariance()
>>> scaler.fit_transform([[0, 0, 0, 0]])
array([[[0.],
        [0.],
        [0.],
        [0.]]])
```

**Expected behavior**
I would expect a similar fix as done in `TimeSeriesScalerMeanVariance`. There, when std=0, the denominator is set to 1 instead (https://github.com/tslearn-team/tslearn/blob/775dadd/tslearn/preprocessing.py#L295). For the `TimeSeriesScalerMinMax`, we should set `range_t` to 1 where it is 0.

**Comment by GillesVandewiele** [comment]
I'll work on this and open a PR in a bit.

**Comment by rtavenar** [comment]
Fixed by #315 
Thanks @GillesVandewiele ;)

**Comment by GillesVandewiele** [comment]
Most welcome, it was a low hanging fruit! Hopefully I can get the other ones in my TODO-list done soon.

Btw, this issue should close itself as soon as `subfolders` branch is merged into `master`.
